packages/react-union/src/constants.js

```diff
@@ -0,0 +1,7 @@
+export const INVALID_JSON = 'INVALID_JSON';
+
+export const IS_SERVER = Boolean(
+	typeof process !== 'undefined' && process.versions && process.versions.node
```

Comment:
can't use just `!process` instead of checking for undefined?

Reply:
Oh yeah, I just kind of copy-pasted this from stack overflow. I'll fix it, sorry.